iscsi/targetd/push_to_cdk

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Should remove push_to_cdk, push_to_dockerhub, build, & build_container since now the Makefile does duplicate things and you've edited the yamls to point to  `quay.io/external_storage/iscsi-controller:latest` instead of `affaelespazzoli/iscsi-controller:0.0.1`